connectors/slack/template/index.js

```diff
@@ -1,24 +1,295 @@
+const Url = require('url');
 const configure = require('./configure');
 const Sdk = require('@fusebit/add-on-sdk');
+const { verifyRequestSignature } = require('@slack/events-api');
+const storage = require('./storage');
+const Superagent = require('superagent');
 
-/*
+/**
+ * @param ctx {FusebitContext}
+ */
+module.exports = (ctx, cb) => {
```

Comment:
Curious - why not async for consistency with other code we've given them?

Reply:
This started as the Express sample, which was written in callback style, and it got inherited once I pulled out Express. I've refactored as async/await.